packages/dynamite/dynamite/lib/src/models/type_result/enum.dart

```diff
@@ -15,9 +15,16 @@ class TypeResultEnum extends TypeResult {
   String? get _builderFactory => null;
 
   @override
-  bool operator ==(Object other) =>
-      other is TypeResultEnum && other.className == className && other.generics == generics && other.subType == subType;
+  TypeResultEnum get dartType => TypeResultEnum(
+        className,
+        subType.dartType,
+        nullable: nullable,
+        isTypeDef: isTypeDef,
+      );
 
   @override
-  int get hashCode => className.hashCode + generics.hashCode + subType.hashCode;
+  bool operator ==(Object other) => other is TypeResultEnum && other.className == className && other.subType == subType;
```

Comment:
Why are the generics no longer compared?

Reply:
they shouldn't have been there from the beginning as an enum can not have any generics